app/models/arclight/parent.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Arclight
```

Comment:
I have no objections to this, but I recall that `DigitalObject` was placed in `app/overrides` (please see https://github.com/pulibrary/pulfalight/blob/master/app/overrides/arclight/digital_object_override.rb). I would just please prefer that one approach be used consistently.

Reply:
Done.